### PR TITLE
Include entire snippets

### PR DESCRIPTION
**Pull request by kalexand-rh**

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@kalexand-rh Does this just apply to snippets, or to modules as well? I thought it was actually a module file that was including only certain lines that broke the builds last week.

File: contributing_to_docs/doc_guidelines.adoc [hunk 1]
```diff
@@ -62,9 +62,10 @@ After the heading block and a single whitespace line, you can include any conten
 
 [NOTE]
 ====
-The assembly title, which is the first line of the document, is the only level 1 ( = ) title.
+* The assembly title, which is the first line of the document, is the only level 1 ( = ) title.
```
[the remark this answers is not in the document]
Comment (bergerhoffer) [reply contributing_to_docs/doc_guidelines.adoc:65]: @kalexand-rh other than capitalizing AsciiDoc, this LGTM!

**Review by opayne1 — COMMENTED** [review]
/lgtm

**Comment by rohennes** [comment]
Is this one ready to merge? I am asking because I have a [Vale rule](https://github.com/redhat-documentation/vale-at-red-hat/pull/590) that flags line ranges in includes ready to go if this advice is still good.

File: contributing_to_docs/doc_guidelines.adoc [hunk 2]
```diff
@@ -62,9 +62,10 @@ After the heading block and a single whitespace line, you can include any conten
...
 Section headers within the assembly must be level 2 ( == ) or lower. When you include modules, you must add
 leveloffsets in the include statements. You can manually add more level 2 or lower section headers in the assembly.
+* You can include only entire modules in assemblies. Although it is valid AsciiDoc, including only link:https://docs.asciidoctor.org/asciidoc/latest/directives/include-lines/#specifying-line-ranges[certain lines] of files can cause build errors and other unintended behavior.
```
Comment (aireilly) [inline contributing_to_docs/doc_guidelines.adoc:68]: Are line ranges allowed in any circumstances? I'm guessing not. 

```suggestion
* You can include only entire modules in assemblies. Although it is valid AsciiDoc, including only link:https://docs.asciidoctor.org/asciidoc/latest/directives/include-lines/#specifying-line-ranges[certain lines] of files can cause build errors and other unintended behavior. Line ranges are not allowed for any included files.
```

**Pull request closed without merging**